### DISCUSSION TITLE
Tools that need modules

### DISCUSSION THREAD

**Discussion by ResistanceIsUseless** [pry0cc/axiom#419] [General]
- [ ] aiodnsbrute -   duplicated install
- [ ] anew
- [ ] anti-burl
- [ ] chaos
- [ ] concurl
- [ ] dnsgen
- [ ] dnsvalidator
- [ ] DumpsterDiver
- [ ] emailgen
- [ ] fprobe
- [ ] getJS
- [ ] gf
- [ ] gron
- [ ] http2smugl
- [ ] interactsh
- [ ] Interlace - NA?
- [ ] kxss
- [ ] leaky-paths
- [ ] proxychains-ng - NA?
- [ ] PwnFile
- [ ] qsreplace
- [ ] Responder
- [ ] scrying
- [ ] sn0int
- [ ] subgen
- [ ] subjack
- [ ] unfurl
- [ ] urlprobe
- [ ] waybackurls
- [ ] zmap
- [X] Amass
- [X] aquatone
- [X] Arjun
- [X] assetfinder -duplicated install
- [X] cf-check
- [X] cngo
- [X] commix
- [X] Corsy
- [X] crlfuzz
- [X] crobat
- [X] ctfr
- [X] dalfox
- [x] dirdar
- [x] DNSCewl
- [x] dnsrecon
- [x] dnsx
- [x] exec
- [x] feroxbuster
- [x] fff
- [x] ffuf - 2 modules
- [x] ffuz
- [x] Finddomain
- [x] gau
- [x] gauplus
- [x] github-endpoints
- [x] github-subdomains
- [x] gobuster - 2 modules
- [x] gorgo
- [x] gospider  - 2 modules
- [x] gowitness
- [x] Gxss
- [x] hakrawler
- [x] httprobe
- [x] httpx
- [x] jaeles
- [x] kiterunner - 3 modules
- [x] linkfinder
- [x] masscan
- [x] massdns
- [x] meg
- [x] naabu
- [x] nmap  - 2 modules
- [x] nuclei
- [x] openredirect
- [x] ParamSpider
- [x] puredns - 3 modules
- [x] Rustscan
- [x] s3scanner
- [x] shuffledns
- [x] soxy
- [x] sqlmap
- [x] subfinder
- [x] subjs
- [x] testssl
- [x] tlscout
- [x] unimap
- [x] wafw00f
- [x] waybackurls
- [x] webscreenshot
- [x] whois
- [x] wpscan

**Comment by 0xtavian** [comment]
@ResistanceIsUseless This list really came in handy when doing this work https://github.com/pry0cc/axiom/blob/master/images/provisioners/full.json. Thank you! <3

**Reply by ResistanceIsUseless** [reply]
Sure thing! Glad i could help.
